bokehjs/src/lib/model.ts

```diff
@@ -23,6 +23,7 @@ export interface Model extends Model.Attrs {}
 
 export class Model extends HasProps {
   properties: Model.Props
+  private _js_callbacks: {[key: string]: any[]}
```

Comment:
Replacing `any[]` with `undefined[]` would be an improvement, and replacing it with `(() => void)[]` would be a greater improvement. In general, we are going away from using `any`, which is unsafe, and replace it with `undefined`. At some point referencing `any` will be disallowed completely.